labs/in-cluster/deploy/k6.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: k6
+  namespace: k6
+  labels:
+    app.kubernetes.io/name: k6
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "9090"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: k6
+  template:
+    metadata:
+      labels:
+        app: k6
+    spec:
+      containers:
+        - name: app
+          env:
+          - name: "K6_PROMETHEUS_RW_SERVER_URL"
+            value: "http://prometheus-service.monitoring.svc.cluster.local:8080/api/v1/write"
+          - name: "MY_HOSTNAME"
+            value: "ngsa-memory.ngsa.svc.cluster.local:8080"
+          image: docker.io/grafana/k6
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
+            runAsNonRoot: true
+            runAsUser: 10001
+          ports:
+            - name: http
+              containerPort: 8080
```

Comment:
Does k6 expose an endpoint at this port? If not, this section and the k6 Service at the bottom of the file can be removed. I did a quick check and it didn't look like anything was responding at this port. I may have missed something though.

Reply:
good point , I will remove it